app/src/sharedTest/java/org/oppia/android/app/parser/ListItemLeadingMarginSpanTest.kt

```diff
@@ -566,22 +573,21 @@ class ListItemLeadingMarginSpanTest {
       htmlResult.getSpanStart(bulletSpan3),
       htmlResult.getSpanEnd(bulletSpan3),
       true,
-      null
+      mock(Layout::class.java)
     )
 
-    val shadowCanvas = shadowOf(canvas)
     // The below assertion verifies the 1st item of inner level list
-    assertThat(shadowCanvas.getDrawnCircle(0).centerX).isWithin(1e-5f).of(-73.0f)
+    assertThat(shadowCanvas.getDrawnCircle(0).centerX).isWithin(1e-5f).of(887.0f)
     assertThat(shadowCanvas.getDrawnCircle(0).centerY).isWithin(1e-5f).of(48.0f)
     // The below assertion verifies the 2nd item of inner level list
-    assertThat(shadowCanvas.getDrawnCircle(1).centerX).isWithin(1e-5f).of(-73.0f)
+    assertThat(shadowCanvas.getDrawnCircle(1).centerX).isWithin(1e-5f).of(887.0f)
     assertThat(shadowCanvas.getDrawnCircle(1).centerY).isWithin(1e-5f).of(139.0f)
 
     // The below assertion verifies the 1st item of outer level list
```

Comment:
Are these comments correct? Per the x values, it seems like this & the next are inner values.

Reply:
It seems to be correct. `x` starting point is `960` so with indentation, outer lists  starts at `935` and inner level starts at `887` point